monetio/models/raqms.py

```diff
@@ -0,0 +1,111 @@
+# Reader for RAQMS real-time files.
+
+import xarray as xr
+
+
+def open_dataset(fname):
+    """Open a single dataset from RAQMS output. Currently set for netcdf file format.
+    Parameters
+    ----------
+    fname : string
+        Filename to be opened.
+    Returns
+    -------
+    xarray.Dataset
+    """
+    names, netcdf = _ensure_mfdataset_filenames(fname)
+    try:
+        if netcdf:
+            f = xr.open_dataset(names[0], drop_variables=["theta"])
+            f = _fix_grid(f)
+            f = _fix_time(f)
+        else:
+            raise ValueError
+    except ValueError:
+        print(
+            """File format not recognized. Note that files should be
+                preprocessed to netcdf."""
+        )
+    # if 'latitude' not in f.coords: print('coordinate issue')
+    return f
+
+
+def open_mfdataset(fname):
+    """Open a multiple file dataset from RAQMS output.
+    Parameters
+    ----------
+    fname : string
+        Filenames to be opened
+    Returns
+    -------
+    xarray.Dataset
+    """
+    names, netcdf = _ensure_mfdataset_filenames(fname)
+    try:
+        if netcdf:
+            f = xr.open_mfdataset(
+                names, concat_dim="time", drop_variables=["theta"], combine="nested"
+            )
+            f = _fix_grid(f)
+            f = _fix_time(f)
+        else:
+            raise ValueError
+    except ValueError:
+        print(
+            """File format not recognized. Note that files should be in netcdf
+                format. Do not mix and match file types."""
+        )
```

Comment:
```suggestion
        print(
            "File format not recognized. Note that files should be "
            "in netCDF format. "
            "Do not mix and match file types."
        )
```
otherwise unintended indentation

Reply:
trim trailing whitespace.................................................Passed
fix end of files.........................................................Passed
check docstring is first.................................................Passed
check yaml...........................................(no files to check)Skipped
pyupgrade................................................................Passed
isort....................................................................Passed
black....................................................................Failed
- hook id: black
- files were modified by this hook

reformatted monetio/models/raqms.py

All done! ✨ 🍰 ✨
1 file reformatted.

black formatter combined some of the lines